src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -788,7 +788,7 @@ impl CatalogState {
                 }
             };
 
-        let table = self.resolve_builtin_table(&MZ_STORAGE_USAGE);
+        let table = self.resolve_builtin_table(&MZ_STORAGE_USAGE_BY_SHARD);
         let object_id_val = match object_id {
```

Comment:
Nit: Could rename the `object_id` vars here to `shard_id` as well, but nbd

Reply:
Good callout, thanks! I tightened up this whole function.